### PR TITLE
update nextcloud from 18.x to 19.0

### PR DESCRIPTION
**Pull request by LiquidITGuy**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] Documentation field contains a link to a page with proper explanations on environmental variables, volumes or the docker compose file.

### CODE REVIEW

File: public/v2/apps/nextcloud.json [hunk 1]
```diff
@@ -81,5 +82,10 @@
         "label": "admin password",
         "description": "Password for the Nextcloud admin user.",
         "validRegex": "/.{1,}/"
+    },{
+        "id": "$$cap_http_https_cors",
+        "label": "will you force https ?",
+        "description": "cors configuration to login are set by the docker image, if you want to use https, please set as https, if not just set as http",
```
Comment (githubsaturn) [inline public/v2/apps/nextcloud.json:88]: A few things here:
1- Change the label to "Protocol of proxy"
2- Add this to the description: `Choose either http or https`
3- Let's add a default value to make it easier for the end user. Perhaps `http`.

Comment (LiquidITGuy) [reply public/v2/apps/nextcloud.json]: Done.

I also add a little informations about nginx optimisation for configuration compliance.
I'll let you checking that :)

**Comment by LiquidITGuy** [comment]
Capture d’écran 2020-06-22 à 23 26 43
Capture d’écran 2020-06-22 à 23 27 05

**Comment by githubsaturn** [comment]
Thanks

**Pull request merged**